svg/global_attributes.json

```diff
@@ -1,6 +1,50 @@
 {
   "svg": {
     "global_attributes": {
+      "autofocus": {
+        "__compat": {
+          "spec_url": "https://svgwg.org/svg2-draft/struct.html#autofocusattribute",
+          "tags": [
+            "web-features:autofocus"
+          ],
+          "support": {
```

Comment:
In general, I'm thinking that there could be a mechanism to mirror the support statement of another feature, to avoid these from becoming inconsistent. This could be bi-directional, i.e. one feature specifies `mirror_to` and the other one specifies `mirror_from`, to make sure editors consider the effect when updating the support statement that is being mirrored from.

Reply:
Agree. This will surely be beneficial. But the way to approach it should be  more discussed.